src/models/db.js

```diff
@@ -117,6 +119,26 @@ export default config => {
     });
   };
 
+  const getArg = arg => {
+    if (arg.type === 'literal') {
+      const literal = db.sequelize.literal('\'' + arg.value + '\'');
+      return db.sequelize.fn(mapFunctions.geography, literal);
+    }
+    return db.sequelize.col(arg.name);
+  };
+
+  const getWhere = filter => {
+    if (filter.type && filter.type === functionCall) {
+      const arg0 = getArg(filter.args[0]);
+      const arg1 = getArg(filter.args[1]);
```

Comment:
There is only 1 with more than 2, which I don't think we are going to implement. In any case, I'm planning to open a new bug for supporting more `functioncalls`, so I'll refactor it a little bit for accommodating as many parameters as needed.